data/json/furniture_and_terrain/furniture-terrains.json

```diff
@@ -900,6 +900,56 @@
       ]
     }
   },
+  {
+    "type": "furniture",
+    "id": "f_flagpole",
+    "name": "flagpole",
+    "description": "A 6m high metal flagpole.  You could hoist up a flag here.",
+    "symbol": "F",
+    "color": "light_gray",
+    "move_cost_mod": -1,
+    "coverage": 40,
+    "required_str": -1,
+    "flags": [ "TRANSPARENT", "PLACE_ITEM" ],
+    "max_volume": "2 L",
+    "bash": {
+      "str_min": 30,
+      "str_max": 100,
+      "sound": "metal screeching!",
+      "sound_fail": "clang!",
+      "items": [
+        { "item": "pipe", "count": [ 1, 7 ] },
```

Comment:
I don't think you'd get individual pipes from smashing a pole, more like scrap and maybe metal chunks

Reply:
Took smash results largely from traffic lights, which gives pipes and scrap. But can remove (or reduce?) the pipes and use metal chunks instead.

Reply:
Replaced pipes by steel chunks (6-15), increased amount of scrap.